src/lib/homepage/Navbar.svelte

```diff
@@ -9,7 +10,18 @@
 		Platform: '/platform'
 	};
 
+	// TODO: shouldn't this work because of the subscription?
+	$: currentPage = $page.url.pathname;
+
 	let menuOpen = false;
+
+	const openMenu = () => {
```

Comment:
Perhaps we should use a component from headless-ui or melt-ui to make the menu accessible?